kotlin-native/Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -385,8 +385,8 @@ private fun processCLib(flavor: KotlinPlatform, cinteropArguments: CInteropArgum
         KotlinPlatform.NATIVE -> {
             val outLib = File(nativeLibsDir, "$libName.bc")
             val compilerCmd = arrayOf(compiler, *compilerArgs,
-                    "-emit-llvm", "-c", outCFile.absolutePath, "-o", outLib.absolutePath)
-            runCmd(compilerCmd, verbose)
+                    "-emit-llvm", "-x", "c", "-c", "-", "-o", outLib.absolutePath)
```

Comment:
The `outCFile` (despite its name) is not always in C, sometimes it is in Objective-C or C++, see the variable definition.

Reply:
Thanks for catching this.

Built the macOS platform libs and the Skia interop test locally - but any other testing suggestions are welcome.

Reply:
Thanks! I'll check the CI anyway, that should be enough.